packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/LineChartDemo/LineChartGreenDemo.tsx

```diff
@@ -1,19 +1,31 @@
 import React from 'react';
-import { Chart, ChartAxis, ChartGroup, ChartLegend, ChartLine, ChartThemeColor } from '@patternfly/react-charts';
-import { VictoryZoomContainer } from 'victory';
+import { Chart, ChartAxis, ChartGroup, ChartLine, ChartThemeColor, ChartLegend, ChartVoronoiContainer } from '@patternfly/react-charts';
+import { width } from 'styled-system';
 
-export class LineChartGreenZoomDemo extends React.Component {
+export class LineChartGreenDemo extends React.Component {
 
   componentDidMount() {
     window.scrollTo(0, 0);
   }
 
   render() {
     return (
-    <div>
-      <div className="line-chart-inline">
-        <div className="line-chart-container">
-          <Chart width={1000} themeColor={ChartThemeColor.green} containerComponent={<VictoryZoomContainer zoomDimension="x" />}>
+      <div style={{width: '70%'}}>
+        <div className="line-chart-legend-right">
```

Comment:
If you cannot get the className working, please remove it. 

Instead of `width: 70%`, please provide the same height and width represented by the className. That way, the charts will scale correctly. The charts and tooltips are still much larger than indented.